services/QuillLMS/client/app/bundles/Staff/components/locker/teamLocker.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import { Link, withRouter } from 'react-router-dom';
+
+import Locker from './locker';
+
+import { lockerItems } from "../../helpers/locker/lockerItems";
+import { titleCase } from "../../../Shared";
+
+const arrow = `${process.env.CDN_URL}/images/icons/arrow-back.svg`;
+
+export const TeamLocker = ({ match }) => {
+  const { params } = match;
+  const { team } = params;
+  const lockerContents = lockerItems[team];
+  const { label, lockers } = lockerContents;
+
+  function renderTeamLockersSection() {
+    return Object.keys(lockers).map(lockerSection => {
+      return(
+        <div className="locker-section-container" key={lockerSection}>
+          <h4>{titleCase(lockerSection)}</h4>
+          <div className="lockers-container">
+            {renderLockerSection(lockerSection)}
+          </div>
+        </div>
+      );
+    });
+  }
+
+  function renderLockerSection(lockerSection: string) {
+    const lockersForSection = lockers[lockerSection];
+    return lockersForSection.map((lockerForSection: string) => {
+      const lockerContents = lockerItems[lockerForSection];
+      return <Locker key={lockerForSection} lockerContents={lockerContents} />
+    });
+  }
+
+  function renderHeader() {
+    if(label === 'all lockers') { return <h3 className="subheader">Index</h3>}
+    return <h3 className="subheader">{`${titleCase(label)} lockers`}</h3>
+  }
+
+  return(
+    <div className="team-locker-container locker-content">
+      <Link className="back-nav-link" tabIndex={-1} to="/locker">
+        <button className="button-container interactive-wrapper focus-on-light">
```

Comment:
Can't we just apply this class to the `<Link>`? This is an internal tool so it isn't a huge deal, but it's an accessibility issue to have a negative tabindex on an interactive element and no interaction on a `button` element, and we might one day have someone on staff who uses a screenreader.

Reply:
good call-- I didn't realize the negative tab index would mess with that. I'll remove the `Link` element and just add an `onClick` handler on the button so we can use the accessible focus indicator on the button